api/src/test/java/org/pmiops/workbench/api/CohortBuilderControllerTest.java

```diff
@@ -103,10 +115,24 @@ public void setUp() {
             mySQLStopWordsProvider);
     controller =
         new CohortBuilderController(
-            cdrVersionService, elasticSearchService, configProvider, cohortBuilderService);
+            elasticSearchService, configProvider, cohortBuilderService, workspaceAuthService);
 
     MySQLStopWords mySQLStopWords = new MySQLStopWords(Arrays.asList("about"));
     doReturn(mySQLStopWords).when(mySQLStopWordsProvider).get();
+    DbCdrVersion cdrVersion = new DbCdrVersion();
+    cdrVersion.setCdrVersionId(1l);
+    DbWorkspace dbWorkspace = new DbWorkspace();
+    dbWorkspace.setWorkspaceNamespace(WORKSPACE_NAMESPACE);
+    dbWorkspace.setName("Saved workspace");
+    dbWorkspace.setFirecloudName(WORKSPACE_ID);
+    dbWorkspace.setCdrVersion(cdrVersion);
+    when(workspaceAuthService.enforceWorkspaceAccessLevel(
+            WORKSPACE_NAMESPACE, WORKSPACE_ID, WorkspaceAccessLevel.READER))
+        .thenReturn(WorkspaceAccessLevel.READER);
```

Comment:
We shouldn't need to mock this, since it isn't called in the CohortBuilderController